### PR TITLE
Move to `go mod`

### PR DESCRIPTION
**Pull request by kpflum-lob**

### CODE REVIEW

**Comment by adrianfalleiro** [comment]
@kpflum-lob Should also bump the version in `.go-version` also?

**Comment by kpflum-lob** [comment]
@adrianfalleiro it is bumped

> @kpflum-lob Should also bump the version in `.go-version` also?

File: echo_test.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func TestMiddleware(t *testing.T) {
 		err := json.Unmarshal([]byte(out), &data)
 		require.NoError(tt, err)
 
-		assert.Equal(tt, "2.2.2.2", data["ip_address"])
+		assert.NotContains(tt, data, "ip_address")
```
Comment (tlhunter) [inline echo_test.go:80]: Obsoletes #11

**Comment by tlhunter** [comment]
LGTM but a repo owner / Go expert should approve.

**Review by senechko — APPROVED** [review]

**Pull request merged**